import-service/serverless.ts

```diff
@@ -0,0 +1,78 @@
+import type { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: 'import-service',
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true
```

Comment:
btw, you can reduce bundle size so:
```
forceExclude:
            - aws-sdk
```

Reply:
@kotmatpockuh thanks a lot! I didn't know about this

Reply:
ps: or you can move  "aws-sdk": "^2.790.0", insidde dev.deps section 👍